realis.go

```diff
@@ -795,16 +825,50 @@ func (r *realisClient) StartJobUpdate(updateJob *UpdateJob, message string) (*au
 		true,
 		func() (*aurora.Response, error) {
 			return r.client.StartJobUpdate(context.TODO(), updateJob.req, message)
-		})
+		},
+		func() (*aurora.Response, bool) {
```

Comment:
can you add some description for this function?
I think its better to define the function elsewhere and call it here.

Reply:
Can't do that because it needs access to the data included in the function that it's verifying.